### DISCUSSION TITLE
Pattern Projector Plus

### DISCUSSION THREAD

**Discussion by j-huff** [Pattern-Projector/pattern-projector#87] [Show and tell]
Hello all,

I've forked Pattern Projector into [Pattern Projector Plus](https://github.com/Pattern-Projector-Plus/pattern-projector-plus). PPP is going to diverge pretty significantly from Pattern Projector, as it employs a simple API server as well [(Pattern Projector Plus API)](https://github.com/Pattern-Projector-Plus/pattern-projector-plus-api).

Right now, the only difference is saving and loading configuration settings, which I mostly did as a proof of concept for the API server. Obviously, this could be achieved by the localstorage mechanism used by Pattern Projector, but more significant features are planned such as basic file storage and organization as well as automatic file conversion (DXF to PDF in particular).

If anyone's interested in contributing, feel free. If you are interested in the multiple calibration settings, I'm sure it could be converted to work with localstorage and added to Pattern Projector pretty easily. Cheers.

**Comment by courtneypattison** [comment]
Thank you for sharing! I haven’t been able to sit at my computer to try it out yet, but I will once I can. I’m excited that you’ll be getting DXF support! I love open source.
